src/ch05-02-example-structs.md

```diff
@@ -158,62 +167,71 @@ struct Rectangle {
 fn main() {
     let rect1 = Rectangle { width: 30, height: 50 };
 
-    println!("rect1 is {}", rect1);
+    println!("rect1 to {}", rect1);
 }
 ```
 
-<span class="caption">Listing 5-11: Attempting to print a `Rectangle`
-instance</span>
+<span class="caption">Listing 5-11: Próba wyświetlenia instancji `Rectangle` </span>
 
-When we run this code, we get an error with this core message:
+Podczas próby uruchomienia tego kodu wyświetlany jest błąd z poniższym komunikatem:
 
 ```text
 error[E0277]: the trait bound `Rectangle: std::fmt::Display` is not satisfied
 ```
 
-The `println!` macro can do many kinds of formatting, and by default, the curly
-brackets tell `println!` to use formatting known as `Display`: output intended
-for direct end user consumption. The primitive types we’ve seen so far
-implement `Display` by default, because there’s only one way you’d want to show
-a `1` or any other primitive type to a user. But with structs, the way
-`println!` should format the output is less clear because there are more
-display possibilities: Do you want commas or not? Do you want to print the
-curly brackets? Should all the fields be shown? Due to this ambiguity, Rust
-doesn’t try to guess what we want, and structs don’t have a provided
-implementation of `Display`.
-
-If we continue reading the errors, we’ll find this helpful note:
+Makro `println!` może formatować na wiele sposobów, a domyślnie para nawiasów klamrowych
+daje `println!` znać, że chcemy wykorzystać formatowanie `Display` (ang. wyświetlenie).
+Jest to wyjście (*ang. output*) przeznaczone dla bezpośredniej konsumpcji przez
+docelowego użytkownika.
```

Comment:
To zdanie jest niezrozumiałe

Reply:
Po pierwsze, skróciłbym do: `Jest to tekst przeznaczony dla docelowego użytkownika`.
Jeśli to nadal jest nie zrozumiałe i chodzi o `docelowego użytkownika` to nie mam pomysłu.

Reply:
Lepiej ;) ta bezpośrednia konsumpcja była dziwna